Stratis.Bitcoin/RPC/Controllers/WalletRPCController.cs

```diff
@@ -41,39 +29,28 @@ public IWalletManager WalletManager
         [ActionName("sendtoaddress")]
         public uint256 SendToAddress(BitcoinAddress bitcoinAddress, Money amount)
         {
+            var account = GetAccount();
             return uint256.Zero;
         }
 
         [ActionName("generate")]
         public List<uint256> Generate(int nBlock)
         {
             var mining = this.serviceProvider.GetRequiredService<PowMining>();
-            var wallet = GetWallet();
-            var address = this.WalletManager.GetUnusedAddress(wallet.WalletName, wallet.Account.Name);
+            var account = GetAccount();
+            var address = this.WalletManager.GetUnusedAddress(account);
             return mining.GenerateBlocks(new ReserveScript(address.Pubkey), (ulong)nBlock, int.MaxValue);
         }
 
-        private UsedWallet GetWallet()
+
+        private WalletAccountReference GetAccount()
         {
+            //TODO: Support multi wallet like core by mapping passed RPC credentials to a wallet/account
```

Comment:
The goal of RPC controller is to be compatible with what has been developped for core. Else I would use `WalletController` 
Core is using RPC creds for mapping to right wallet.